libraries/chain/block_log.cpp

```diff
@@ -532,60 +497,21 @@ namespace eosio { namespace chain {
    }
 
    void block_log::reset(const genesis_state& gs, const signed_block_ptr& first_block, uint32_t first_block_num) {
-       EOS_ASSERT(first_block_num == 1, block_log_exception,
```

Comment:
This change doesn't look consistent.
Blocklog doesn't support starting from other block than 1.
For what this change?